autogluon/training/docker/0.6/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,61 @@
+ARG PYTHON_VERSION=3.8.10
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-training:1.9.0-cpu-py38-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+ENV MXNET_CUDNN_LIB_CHECKING=0 \
+    MXNET_CUDNN_AUTOTUNE_DEFAULT=0 \
+    PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Install dependencies
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get install -y --no-install-recommends \
+    libncurses5-dev \
+    libnss3-dev \
+    libgl1-mesa-glx \
+    liblzma-dev \
+    uuid-dev \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG AUTOGLUON_VERSION=0.6.1
+ARG TORCH_VER=1.12.1+cpu
+ARG TORCH_VISION_VER=0.13.1+cpu
+ARG TORCH_TEXT_VER=0.13.1
+
+# NOTE: reinstalling horovod with pytorch support because it is available in the container
+# Updating certifi - https://pyup.io/p/pypi/certifi/2ff/
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools wheel \
+ && pip install --no-cache-dir -U numpy numba \
+ && pip install --no-cache-dir -U certifi \
+    torch==${TORCH_VER} torchvision==${TORCH_VISION_VER} torchtext==${TORCH_TEXT_VER} --extra-index-url https://download.pytorch.org/whl/cpu \
+ && pip uninstall -y horovod \
+ && HOROVOD_WITH_PYTORCH=1 pip install --no-cache-dir horovod==0.24.3 \
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+ && mim install -q mmcv-full \
+ && pip install --no-cache-dir -U mmdet
```

Comment:
@gradientsky Would you like to add integration tests for mmcv-full and mmdet? Are integration tests needed here for these newly added features?

Reply:
These are transparent for the users when they use CV models, it's a backend change (similar how the transition from mxnet-based NNs to pytorch happened). mim is a model zoo for object detection models.

Reply:
Thanks! Resolving conversation on this comment.